src/python/WMCore/Credential/Proxy.py

```diff
@@ -174,13 +174,16 @@ def setEnv(self, cmd):
     def cmd_exists(self, cmd):
         return subprocess.call(self.setEnv("type " + cmd), shell=True,
                                stdout=subprocess.PIPE, stderr=subprocess.PIPE) == 0
+    def getUserCertFilename(self):
+        ucFilename = '~/.globus/usercert.pem' if 'X509_USER_CERT' not in os.environ else os.environ['X509_USER_CERT']
```

Comment:
I'd suggest to expand it to multiple lines in order to make it more readable.

Reply:
Done.